setup.cfg

```diff
@@ -1,3 +1,61 @@
+[metadata]
+name = jira_timemachine
+version = 0.0.0
+url = https://github.com/ClearcodeHQ/jira_timemachine
+description = Synchronize worklogs between Jira instances
+long_description = file: README.rst, CHANGES.rst
+long_description_content_type = text/x-rst
+keywords = jira, worklog, timesheet
+license = MIT License
+author = Grzegorz Śliwiński
+author_email = fizyk+pypi@fizyk.net.pl
+maintainer = Michał Masłowski
+maintainer_email = m.maslowski@fizyk.net.pl
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+
+[options]
+zip_safe = False
+include_package_data = True
+python_requires = >= 3.8
+packages = find:
+package_dir =
+    =src
+install_requires =
+    arrow>=1.1.0
+    attrs>=21.2.0
+    Click>=8.0.1
+    types-click>=7.1.0
+    jira>=3.0.1
+    typing>=3.7.4.3
+    requests>=2.25.1
+    types-requests>=0.1.9
+
+[options.packages.find]
+where = src
+
+[options.extras_require]
+tests =
+    pytest
+    pytest-cov
+    mock
```

Comment:
Not an issue for this PR, but we might want to check which dependencies are still needed; this one can be replaced by `unittest.mock`.